src/libs/routes/errorHandler.ts

```diff
@@ -0,0 +1,13 @@
+import {Request, Response, NextFunction } from 'express'
+
+export default ((err, req : Request, res : Response, next : NextFunction) => {
+    console.log(err);
+    res.json(
+        {
+            error : err.error,
+            status : err.code,
+            message : err.message || "Error",
+            timeStamp: new Date()
+        }
```

Comment:
Give proper name to function

Reply:
I forgot to add function but I will give proper function name  in this particular files.